scripts/events_module/short/condition_events.py

```diff
@@ -318,8 +318,7 @@ def handle_illnesses(cat, season=None):
                     "stomachache",
                 ]:
                     illness = f"a {chosen_illness}"
-                # try to translate the illness
-                illness = i18n.t(f"conditions.illnesses.{chosen_illness}")
+
                 illness.replace("conditions.illnesses.", "")
```

Comment:
```suggestion
```
If you're intending to remove the translation attempt of an unrecognised condition, also remove this line.

Reply:
Ah, I did not, must've been a merge quirk. I'll put it back to how it was.